server/src/visitors/YmlClassVisitor.ts

```diff
@@ -22,6 +27,14 @@ export class YmlClassVisitor extends YmlToBdlBaseVisitor {
       CompletionItemKind.Method,
       this.classId,
     );
+    this.definitions.addDefinition(
+      createLocation(
+        node.methodIntro().ymlId().text,
+        node.start,
+        node.stop,
+        this.uri,
+      ),
```

Comment:
Surprised by this trailing comma since it does not seem to be a list or anything, and the function only seems to accept one location at a time anyway. :thinking:

Reply:
Autoformatting. Seems to be a good practice in TS :open_mouth:

Reply:
Oh, I guess it's for when you refactor and add an argument to the method. Then it behaves kinda like when you add items to a list or array.